Could not find a declaration file for module 'luxon'

### Description

The following error message is thrown from the cron module (using Typescript):

```
node_modules/cron/types/index.d.ts:11:26 - error TS7016: Could not find a declaration file for module 'luxon'. '/api/node_modules/luxon/build/node/luxon.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/luxon` if it exists or add a new declaration (.d.ts) file containing `declare module 'luxon';`
```

The issue is caused by a missing type definition for the luxon library. The [docs](https://github.com/moment/luxon/blob/master/docs/install.md#types) for the library states it should be installed as a developer dependency separately for typescript. 

### Screenshots

<img width="633" alt="Screenshot 2023-08-16 at 11 58 11" src="https://github.com/kelektiv/node-cron/assets/2177941/dc97ee74-4889-44d0-9207-7293c13e9d32">


### Additional information

_No response_

--- a/package.json
+++ b/package.json
@@ -20,7 +20,8 @@
 	},
 	"types": "types/index.d.ts",
 	"dependencies": {
-		"luxon": "^3.2.1"
+		"@types/luxon": "~3.3.0",
+		"luxon": "~3.3.0"
 	},
 	"devDependencies": {
 		"@commitlint/cli": "~17.6.6",

